demo/ThemeTransform.html

```diff
@@ -0,0 +1,203 @@
+<!doctype html>
+<html>
+
+  <head>
+    <meta charset=utf-8>
+    <title>ThemeTransform</title>
+    <style>
+      :root {
+        --control-panel-theme-visibility: visible;
+      }
+      body {
+        padding: 20px 20px 20px 300px !important;
+      }
+      .content_block {
+        padding-bottom: 50px;
+      }
+      #theme_demo_menu a {
+        display: block;
+        color: black;
+        font-size: 0.7em;
+        font-weight: bold;
+        padding: 8px 0px;
+      }
+      #theme_demo_tools {
+        background: #ccc;
+        padding: 15px;
+        position: fixed;
+        left: 20px;
+        top: 20px;
+        z-index: 10000;
+        width: 230px;
+        border-radius: 8px;
+      }
+      .theme_demo_article_title {
+        color: red;
+        font-size: 2.0em;
+      }
+    </style>
+ 
+    <link href="https://en.wikipedia.org/w/load.php?debug=false&lang=en&only=styles&skin=vector&modules=skins.minerva.base.reset|skins.minerva.content.styles|ext.math.styles|" rel="stylesheet" type="text/css"></link>
+
+    <script src=build/wikimedia-page-library-transform.js></script>
+    <script src='./DemoArticles/ArticleRef.js'></script>
+    
+    <link rel=import href=ControlPanel.html>
+  </head>
+
+  <body>
+    <div id='theme_demo_tools'>
+      <control-panel></control-panel>
+      <div id='theme_demo_menu'></div>
+    </div>
+    <div class='content' id='content'></div>
+    
+    <script>
+
+/**
```

Comment:
The indentation for this appears to be incorrect. Any idea why the linter isn't picking this up?